ci/scripts/run-unit-tests.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env sh
+set -euo pipefail
+
+# log current go version
+go version
+
+# install gcc
+apk --no-progress add build-base
```

Comment:
Why is this line needed?

Reply:
build-base contains gcc

Reply:
Hi @rkoster,

This is to fix:
```
...
go: downloading golang.org/x/net v0.8.0
go: downloading golang.org/x/sys v0.6.0
go: downloading gopkg.in/yaml.v3 v3.0.1
go: downloading golang.org/x/text v0.8.0
# runtime/cgo
cgo: C compiler "gcc" not found: exec: "gcc": executable file not found in $PATH

Failed to compile config:

# runtime/cgo
cgo: C compiler "gcc" not found: exec: "gcc": executable file not found in $PATH
```

Reply:
Would compiling with `CGO_ENABLED=0` also work?

Reply:
Ahh nice hint, I updated the PR accordingly.
THX!!